pramen/core/src/main/scala/za/co/absa/pramen/core/utils/ConfigUtils.scala

```diff
@@ -269,6 +269,24 @@ object ConfigUtils {
     }).toMap
   }
 
+  def unwrap(conf: Config): Map[String, AnyRef] = {
```

Comment:
I like the method. It looks very useful, consider putting it to `ConfigUtils`. Maybe as something like `convertToMap()`

Reply:
Great name! I couldn't figure anything out so I named it `unwrap` 😃